app/src/main/res/layout-sw600dp/option_audio_language.xml

```diff
@@ -20,9 +20,9 @@
     android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"
```

Comment:
You need to delete this file and add this extra code to the main file

`android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"`

Reply:
I think it won't be an issue if you delete this file and add code in the main file. So, I would suggest first try to run by removing this file and if it throws an error or UI is changed then you can keep this file.

Reply:
Okay